### PR TITLE
Mario/scale images

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

File: src/mapbox/layers/TrackerLayer.js [hunk 1]
```diff
@@ -31,11 +39,15 @@ class TrackerLayer extends mixin(Layer) {
     if (!map) {
       return;
     }
-    const { width, height } = map.getCanvas();
+    const canvas = map.getCanvas();
+    const iconScale = canvas.width / canvas.clientWidth;
+
+    window.addEventListener('resize', this.updateIconScale.bind(this, canvas));
```
Comment (oterral) [inline src/mapbox/layers/TrackerLayer.js:45]: listening `map.on('change:size')` seems more accurate

**Review by oterral — CHANGES_REQUESTED** [review]
I'm wondering if the auto scaling of the icon could be done in the tracker directly , if not. this implementaion is ok for me.

File: src/common/mixins/TrajservLayerMixin.js [hunk 2]
```diff
@@ -431,7 +431,7 @@ const TrajservLayerMixin = (TrackerLayer) =>
         const origin = radiusDelay + margin;
 
         const canvas = document.createElement('canvas');
-        canvas.width = radiusDelay * 2 + margin * 2 + 100;
+        canvas.width = radiusDelay * 2 + margin * 2;
```
[the remark this answers is not in the document]
Comment (mhaertwig) [reply src/common/mixins/TrajservLayerMixin.js]: Indeed, I fixed it and added a comment.

**Comment by mhaertwig** [comment]
There's no `map` in Tracker.js and the listening to map updates is different in `ol` and `mapbox`. So I would leave the determination of the scale size in the corresponding layers.

**Pull request merged**